CHANGELOG.md

```diff
@@ -1,5 +1,90 @@
+- Don't delete ctl symlink for amazon linux [\#970](https://github.com/chef/chef-server/pull/970) ([srenatus](https://github.com/srenatus))
+- Bump omnibus-software to 59e7d9 [\#967](https://github.com/chef/chef-server/pull/967) ([srenatus](https://github.com/srenatus))
+- Bump omnibus software to b04ac7e2 [\#965](https://github.com/chef/chef-server/pull/965) ([danielsdeleo](https://github.com/danielsdeleo))
+- Do not fail complete reindex when one object fails [\#964](https://github.com/chef/chef-server/pull/964) ([smith](https://github.com/smith))
+- Simple script for analyzing disk space usage [\#963](https://github.com/chef/chef-server/pull/963) ([stevendanna](https://github.com/stevendanna))
+- Update omnibus-software for latest berks & chef improvements [\#962](https://github.com/chef/chef-server/pull/962) ([stevendanna](https://github.com/stevendanna))
+- a ctl-command to dump indexable data to json [\#961](https://github.com/chef/chef-server/pull/961) ([marcparadise](https://github.com/marcparadise))
+- Don't include source in erlang releases [\#958](https://github.com/chef/chef-server/pull/958) ([stevendanna](https://github.com/stevendanna))
+- Make loading reporting in dvm easier [\#956](https://github.com/chef/chef-server/pull/956) ([ksubrama](https://github.com/ksubrama))
+- Remove nodejs from the final package [\#955](https://github.com/chef/chef-server/pull/955) ([stevendanna](https://github.com/stevendanna))
+- Cleanup cached gem files in ruby install [\#954](https://github.com/chef/chef-server/pull/954) ([stevendanna](https://github.com/stevendanna))
+- Bump omnibus-software [\#953](https://github.com/chef/chef-server/pull/953) ([stevendanna](https://github.com/stevendanna))
+- Policy data collector [\#951](https://github.com/chef/chef-server/pull/951) ([danielsdeleo](https://github.com/danielsdeleo))
+- Add cleanup build step [\#950](https://github.com/chef/chef-server/pull/950) ([stevendanna](https://github.com/stevendanna))
+- Install nodejs from binary packages [\#949](https://github.com/chef/chef-server/pull/949) ([stevendanna](https://github.com/stevendanna))
+- Update contributing docs [\#948](https://github.com/chef/chef-server/pull/948) ([tas50](https://github.com/tas50))
+- \[omnibus\] update mixlib-install to 2.0 for PackageRouter support [\#943](https://github.com/chef/chef-server/pull/943) ([wrightp](https://github.com/wrightp))
+
+### Components
+
+**New Components**
+
+* libintl-perl (1.23)
+* nodejs-binary (6.7.0)
+* berkshelf-no-depselector (03b43842afabd1f9c3bf127dafb1dc1dd0daba64)
+* cleanup (1.0.0)
+
+**Updated Components**
+* config\_guess (9152ce40 -> 084a4076)
+* openssl (1.0.1u -> 1.0.2j)
+* mixlib-install (b2495ce9 -> 9c9dad45)
+* openresty (1.9.7.2 -> 1.11.2.1)
+* knife-opc (2ce18240 -> a08237d4)
+* ohai (567dcb84 -> 582dcc7a)
+* appbundler (379a06cc -> 76cb1728)
+* rb-readline (cf67cd06 -> 5e1c55db)
+* chef_backup-gem (cc8e5bf9 -> 67b1f51a)
+* veil-gem (master -> master)
```

Comment:
Hrm, it is interesting that it didn't pick up the sha's here...Not to worried about it for now but might be worth looking at later.